mysql-test/suite/rocksdb_dd_innodb/t/i_s_innodb_tables.test

```diff
@@ -0,0 +1 @@
+--source ../../innodb/t/i_s_innodb_tables.test
```

Comment:
maybe I miss something... do you also need to config default_dd_storage_engine to rocksdb for this test suite?

Reply:
Yes, it will be set once it is viable (the rest of the feature gets merged). Now these tests were developed with the `dd_access_through_server()` hardcoded to return `true`